### DISCUSSION TITLE
公式编号的问题

### DISCUSSION THREAD

**Discussion by GPY1999** [tuna/thuthesis#672] [Q&A] [answered]
本科论文，现在编译出来的pdf里公式编号的(1.1)，但是论文的格式要求是(1-1)，这个有解决方法吗

**Comment by zepinglee** [comment]
本科生的要求：
<img width="664" alt="Screen Shot 2021-05-27 at 11 55 21" src="https://user-images.githubusercontent.com/12290822/119764224-99ac6d80-bee3-11eb-8276-1a0575fb359d.png">

这里出现了“x.x”和“x-x”两种形式的编号，但没有进一步说明。

按照研究生的要求，两种分隔符都是允许的。

<img width="671" alt="Screen Shot 2021-05-27 at 12 10 33" src="https://user-images.githubusercontent.com/12290822/119764745-8bab1c80-bee4-11eb-9c27-511b432bc431.png">

通常来说，编号的分隔符应全文统一。所以目前默认都使用“x.x”的格式。不过也允许用户进行设置。

<img width="722" alt="Screen Shot 2021-05-27 at 12 14 03" src="https://user-images.githubusercontent.com/12290822/119765069-11c76300-bee5-11eb-96de-e8765ae2e856.png">

**Comment by zepinglee** [comment]
有没有必要严格按照示例把默认设置改成“图 2.1”、“表 3.1”、“式（1-2）”？ @Harry-Chen

**Comment by Harry-Chen** [comment, marked as answer]
@GPY1999 此问题是否格式审查提出的？如果就公式不一样，个人看起来有点不舒服。按照往年经验，格式审查并不会在编号分隔符上抠细节。我倾向于维持现状，如果存在格式审查老师有意见，可以通过选项单独进行修改。
